<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GpsdRawDataViewer</class>
 <widget class="QWidget" name="GpsdRawDataViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPSD Raw Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="formatLabel">
       <property name="text">
        <string>For&amp;mat:</string>
       </property>
       <property name="buddy">
        <cstring>rawDataFormat</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="rawDataFormat">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Select data format</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="rawDataClear">
       <property name="toolTip">
        <string>Clear</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="icon">
        <iconset resource="qtgpsc.qrc">
         <normaloff>:/rsc/images/edit-clear.png</normaloff>:/rsc/images/edit-clear.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="rawDataCopy">
       <property name="toolTip">
        <string>Copy to clipboard</string>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
       <property name="icon">
        <iconset resource="qtgpsc.qrc">
         <normaloff>:/rsc/images/edit-copy.png</normaloff>:/rsc/images/edit-copy.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="rawDataSave">
       <property name="toolTip">
        <string>Save to file</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="qtgpsc.qrc">
         <normaloff>:/rsc/images/document-save.png</normaloff>:/rsc/images/document-save.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="rawDataViewer">
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="qtgpsc.qrc">
     <normaloff>:/rsc/images/edit-clear.png</normaloff>:/rsc/images/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear raw data</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="qtgpsc.qrc">
     <normaloff>:/rsc/images/edit-copy.png</normaloff>:/rsc/images/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy raw data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="qtgpsc.qrc">
     <normaloff>:/rsc/images/document-save.png</normaloff>:/rsc/images/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save raw data to file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qtgpsc.qrc"/>
 </resources>
 <connections/>
</ui>
